frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/TeamDetailsWrapper.tsx

```diff
@@ -144,6 +144,7 @@ const TeamDetailsWrapper = (props: ITeamDetailsPageProps): JSX.Element => {
       }
       dispatch(teamActions.update(team?.id, updatedAttrs))
         .then(() => {
+          dispatch(teamActions.loadAll({ perPage: 500 }));
```

Comment:
I'm ok with merging these changes in.

It does seem strange that we have to load all the teams instead of receiving the new name from the `PATCH` response body.

Reply:
Yeah, I agree. In the interest of time, I opted for the quick one-line fix that didn't require getting too deep into the Redux state.